src/executables/sysrepocfg.c

```diff
@@ -930,6 +931,178 @@ srcfg_import_datastore(struct ly_ctx *ly_ctx, int fd_in, md_module_t *module, sr
     return rc;
 }
 
+/**
```

Comment:
`srcfg_merge_datastore()` differs from `srcfg_import_datastore()` just a little, please avoid code duplication and merge the code into a single function with some parameter/option to switch between import and merge functionality. The same applies to `srcfg_merge_operation()` and `srcfg_import_operation()`.